scripts/remove_boilerplate.py

```diff
@@ -80,28 +81,16 @@ def read(self, index_file):
         index_iter = self.index_lines(index_file)
         warc_iter = self.warc_records(index_file)
         index_id = 0
-        errors = 0
         for warc_record in warc_iter:
             url = warc_record['WARC-Target-URI']
             for index in index_iter:
                 index_id += 1
                 if unquote_inf(index.url) == unquote_inf(url):
-                    try:
-                        self.process_record(index_id, index, warc_record)
-                    except:  # noqa
-                        logging.exception(f'Exception in file {index_file} '
-                                          f'on line {index_id} ({url}).')
-                        errors += 1
-                        if errors == 3:
-                            raise ValueError(f'Too many errors ({errors}).')
+                    self.process_record(index_id, index, warc_record)
```

Comment:
I am not sure about deleting this, but OK.

Reply:
Well, it did 2 things:
If only 1-2 errors happened, it swallowed them without even logging.
If at least 3 errors happened, then the program still died, but with zero information about what was the problem.

That's (almost) literaly the worst way you can handle errors!
(Probably the literaly worst way would be to replace the "more than 3" part with a random coin toss...)

Reply:
That's not true, as you'd still have the logging messages that told you _exactly_ what the three errors were. So basically we allowed the scrip to recover from two erroneous entries. Now we fail on the first. Normally I would say this is a regression in functionality.

The reason I used the word "normally" is because I grepped through the logs and found no such messages. So I figure we can except the data to be correct all times, in which case I don't mind failing fast. That's why I said OK in my first comment, even though I am still in two minds about this.